### PR TITLE
chore(cb2-2406): update readme

### PR DESCRIPTION
**Pull request by andrewf-bjss**
Supporting documentation changes ONLY:

- Updated `README.md`
- Updated `env.example`
- Removed `./config/index.ts` as it was copied to `./src/config/index.ts` and original is no longer referenced
- Adds `serverless-dotenv-plugin` plugin

[CB2-2406](https://jira.dvsacloud.uk/browse/CB2-2406)

## Checklist (n/a)
- [ ] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Comment by andrewf-bjss** [comment]
rebased against feature branch (feature/CB2-2097)

File: README.md [hunk 1]
```diff
@@ -1,109 +1,46 @@
...
-
-The application runs on port `:3001` by default when no stage is provided.
+Designed to be invoked by a timer every night to pick up the previous days changes, but can be invoked manually.
 
-To get started, please change the meta data of your `package.json` and `serverless.yml` file accordingly as well as base api route for your express router.
-The router is currently being mounted in the following file `src/infrastructure/api/index.ts`.
+## Description
 
-The service has local environmental variables (please see `env` placeholder file) set locally however should we wish to further extend the service, the environmental variables will need to be ported over to the CI/CD pipeline which currently uses `BRANCH` and `BUCKET`.
+The function authenticates with Azure AD, and uses the returned token to retrieve the details of ATFs that have changed from Dynamics CE using OData. Each updated ATF is added to EventBridge as a separate event.
```
Comment (aaron-downing) [inline README.md]: Might want to change the use of ATF to test station as we're going to be dealing with different types of test stations (including ATFs)

Comment (andrewf-bjss) [reply README.md]: thanks @aaron-downing - I've replaced all references to 'ATF' with 'test station'

**Review by aaron-downing — APPROVED** [review]

**Review by Ian-Hodges — COMMENTED** [review]
I would have kept most of the parts that you have removed. The Code Structure section can definitely go. Parts like the environmental files are important in my opinion.

**Review by ghost — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -126,46 +132,20 @@ To facilitate the standardisation of the code, a few helpers and tools have been
...
-
-#### Code structure
-
-Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers.
-
-<p align="center">
-  <img src="./docs/DDD_architecture.jpg" alt="Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers" >
-</p>
-
-#### Toolings
```
Comment (Ian-Hodges) [inline README.md:157]: I'd keep this section. Especially the sonarqube bit.

Comment (andrewf-bjss) [reply README.md:157]: Kept, thanks @Ian-Hodges - I've added some instructions for running SQ in docker too

**Review by Ian-Hodges — APPROVED** [review]

**Pull request merged**